### DISCUSSION TITLE
on backspace deletion it's delete all thing from terminal including root user also

### DISCUSSION THREAD

**Discussion by ratansinghgit** [sourcelair/xterm.js#4506] [Q&A]
Hi, 

when I delete some character from backspace it's delete everything, please help me out this issue

**Comment by jerch** [comment]
What do you mean by "delete everything" and "including root user"? I hope it did not format your disks... :scream: 

Plz be more specific on:
- your setup
- what you did
- what you expected
- what happened instead
